production/dpl-workflow.sh

```diff
@@ -244,17 +247,27 @@ elif [ $EXTINPUT == 1 ]; then
     if has_detector_flp_processing $i; then
       case $i in
         TOF)
-          PROXY_INTYPE=CRAWDATA;;
+          PROXY_INTYPE="CRAWDATA";;
+        FT0 | FV0 | FDD)
+          PROXY_INTYPE="DIGITSBC/0 DIGITSCH/0";;
+        PHS)
+          PROXY_INTYPE="CELLS CELLTRIGREC";;
+        CPV)
+          PROXY_INTYPE="DIGITS/0 DIGITTRIGREC/0 RAWHWERRORS";;
+        EMC)
+          PROXY_INTYPE="CELLS/0 CELLSTRGR/0 DECODERERR";;
```

Comment:
Better not use subspecs. E.g. EMC gets data from 2 FLPs, so the subspec will likely be the FEEID of the 1st link (they have open PR on hold for merging of such data).

Reply:
Actually, at least for FT0 | FV0 | FDD and CPV the subspec 0 is needed for sure, since downstream devices expect this and there is anyway 1 FLP. For the EMC it is certainly not needed as 2 streams need to be merged. Will need to clarify for the PHOS.